docker/run_jaxbtck.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x
+
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0, which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception, which is available at
+# https://www.gnu.org/software/classpath/license.html.
+#
+# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
```

Comment:
was this file intentionally left blank?

Reply:
This PR only contains the outline changes required for creating pipeline for jaxb build and run. More changes are to be made in these files like docker/run_jaxbtck.sh which will be modified soon to include the run script for jaxb.